code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -263,6 +263,40 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
 		return
 
 	ui_interact(user)
+//Cafe stuff
+
+/obj/machinery/chem_dispenser/brewer
+	name = "Space-Brewery"
+	icon_state = "brewer"
+	var/obj/item/weapon/reagent_containers/glass/kettle = null

```

Comment:
This shouldn't be necessary, and I think it would break code to try and use it because all the reagents are moved between the beaker var of chem_dispenser anyways.